core/trino-main/src/main/java/io/trino/operator/output/PositionsAppenderPageBuilder.java

```diff
@@ -26,6 +28,9 @@
 public class PositionsAppenderPageBuilder
 {
     private static final int DEFAULT_INITIAL_EXPECTED_ENTRIES = 8;
+    @VisibleForTesting
```

Comment:
The tests do assert on this value, since the test logic requires knowing what the value is in order to trigger the flushing based on position count without reaching the size limit.